### PR TITLE
Improve cache line on tab-switcher

### PR DESCRIPTION
**Pull request by subsymbolic**
Reviewer: Caine
Asana: https://app.asana.com/0/361428290920652/386683237338087

**Description**:
Update cached text to be more human friendly. Also updated it to only display when we initially enter the tabs switcher if we have cached data

**Steps to test this PR**:
1. Open the app, browse to a webpage
2. Open the tab switcher and observe new text
3. Press the forget button and note that the text disappears

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR DRI Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications (Database connections, Grafana stats, CPU)

### CODE REVIEW

File: DuckDuckGo/Localizable.strings [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 "tabswitcher.title.tabs" = "Private Tabs";
 "tabswitcher.title.notabs" = "No Tabs";
-"tabswitcher.data" = "%d items cached";
+"tabswitcher.data" = "Tap 'Forget' to erase your tabs and data";
```
Comment (nilnilnil) [inline DuckDuckGo/Localizable.strings]: Looking at these, what's our plan for translating them?

**Comment by nilnilnil** [comment]
Conflicts here -- merge when ready.

File: DuckDuckGo/Localizable.strings (hunk 1 above)
Comment (subsymbolic) [inline DuckDuckGo/Localizable.strings]: We have a separate post-launch translation task in Asana.

**Comment by subsymbolic** [comment]
Travis seems to be having issues, I've validated that the tests all pass so I'm going ahead and merging this.

**Pull request merged**